packages/core-bridge/Cargo.toml

```diff
@@ -30,3 +30,4 @@ once_cell = "1.7.2"
 temporal-sdk-core = { version = "*", path = "./sdk-core/core", features = ["ephemeral-server"] }
 temporal-client = { version = "*", path = "./sdk-core/client" }
 tokio-stream = "0.1"
+dashmap = "5.5"
```

Comment:
Why adding this dependency in bridge? I see core is asking for it (v5.0), but it isn't used in bridge...

Reply:
This is a workaround for a small dependency specification bug in sdk-core. I've opened https://github.com/temporalio/sdk-core/pull/650 and will remove this when it's resolved in sdk-core.